### PR TITLE
Project readme update - update command count reference

### PR DESCRIPTION
**Pull request by 0x7FFFFFFFFFFFFFFF**
```
PS C:\> (gcm -module dbatools | Measure-Object).Count
707
PS C:\>
```

### CODE REVIEW

**Review by niphlod — APPROVED** [review]

**Comment by ClaudioESSilva** [comment]
Hi folks, 
We are still below the 700 mark.
We have: 694 (Functions (692) + CmdLet (2))
And: 13 Alias

**Comment by niphlod** [comment]
`has now grown into a collection of [over 700 commands]` to `has now grown into a collection of [nearly 700 commands]` ;-)

File: readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Getting Started
 
-<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but has now grown into a collection of [over 500 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
+<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but has now grown into a collection of [over 700 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
```
Comment (wsmelton) [inline readme.md:5]:
```suggestion
<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but as of our v2 release has grown into a collection of [over 700 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
```

File: readme.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Getting Started
 
-<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but has now grown into a collection of [over 500 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
+<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but as of our 2.0 version release has grown into a collection of [over 700 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
```
Comment (jpomfret) [inline readme.md:5]:
```suggestion
<img align="left" src=bin/dbatools.png alt="dbatools logo">  dbatools is PowerShell module that you may think of like a command-line SQL Server Management Studio. The project initially started out as just `Start-SqlMigration.ps1`, but as of our 2.0 version release has grown into a collection of [nearly 700 commands](https://dbatools.io/commands) that help automate SQL Server tasks and encourage best practices.
```

**Comment by niphlod** [comment]
dunno what to do except approve all of the above ... let's not keep PRs around for weeks ;-)

**Comment by jpomfret** [comment]
I've committed that slight wording tweak and once the checks pass I'll merge it in - this is an easy one :)

**Pull request merged**